cyences_app_for_splunk/default/data/ui/views/cs_gsuite_reports.xml

```diff
@@ -53,14 +57,34 @@
       </chart>
     </panel>
   </row>
-<!-- TO DO: check below panel's event_name=login_challenge and login_challenge_status field is not present in new TA props-->
+  <row>
+    <panel>
+      <title>Login by Location</title>
+      <map>
+        <search>
+          <query>`cs_gsuite_login_events` "event.name" IN("login_success", "login_failure") user=$tkn_user|s$
+| iplocation ipAddress 
+| geostats count(user) by action</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <option name="drilldown">all</option>
+        <option name="mapping.fieldColors">{"success": #008000, "failure": #ff0000}</option>
+        <option name="mapping.type">marker</option>
+        <option name="refresh.display">progressbar</option>
+        <drilldown>
+          <link target="_blank">search?q=%60cs_gsuite_login_events%60%20%22event.name%22%20IN(%22login_success%22%2C%20%22login_failure%22)%20user%3D$tkn_user|s$%0A%7C%20iplocation%20ipAddress%20%0A%7C%20search%20lat%3E%3D$click.bounds.south$%20lat%3C$click.bounds.north$%20lon%3E%3D$click.bounds.west$%20lon%3C$click.bounds.east$%20%0A%7C%20%60cs_human_readable_time_format(_time%2C%20Time)%60%0A%7C%20rename%20action%20as%20%22Login%20Status%22%2C%20user%20as%20User%2C%20ipAddress%20as%20%22IP%20Address%22%0A%7C%20table%20Time%20%22Login%20Status%22%20User%20%22IP%20Address%22%20City%20Region%20Country&amp;earliest=$timeRange.earliest$&amp;latest=$timeRange.latest$</link>
+        </drilldown>
+      </map>
+    </panel>
+  </row>
   <row>
     <panel>
       <title>Login Challenged</title>
       <table>
         <title>User was asked for extra login question due to suspicious login activity</title>
         <search>
-          <query>`cs_gsuite_login_events` user=$tkn_user|s$ event_name="login_challenge" | sort login_challenge_status | table _time, user, ipAddress, login_type, login_challenge_method, login_challenge_status</query>
+          <query>`cs_gsuite_login_events` user=$tkn_user|s$ "event.name"="login_challenge" | sort login_type | `cs_human_readable_time_format(_time, Time)` | rename user as User, ipAddress as IPAddress, action as Action, login_challenge_method as LoginChallengeMethod, login_type as LoginType | table Time, User, IPAddress, Action, LoginChallengeMethod, LoginType</query>
```

Comment:
login_challenge of google workspace is kind of similar to 2FA or MFA or Azure, so we create alerts and dashboard-panels according to that.

Reply:
Yes, will check. it is kind of security question when you do the suspicious activity then google asks for the verification.

one of the method is by adding the Employee Id as a security question. Rahaf has tried to login from new devices but the question is not coming up. (https://support.google.com/a/answer/6002699?hl=en)